utilities/boostHistHelpers.py

```diff
@@ -211,8 +211,8 @@ def extendHistByMirror(hvar, hnom, downAsUp=False, downAsNomi=False):
 def addGenChargeAxis(h, idx):
     return addGenericAxis(h, hist.axis.Regular(2, -2., 2., underflow=False, overflow=False, name = "qGen"), idx, add_trailing=False, flow=True)
     
-def addSystAxis(h, size=1, offset=0):
-    return addGenericAxis(h, hist.axis.Regular(size,offset,size+offset, name="systIdx"))
+def addSystAxis(h, size=1, offset=0, axname="systIdx"):
+    return addGenericAxis(h, hist.axis.Regular(size,offset,size+offset, name=axName))
```

Comment:
You could use Integer

Reply:
I didn't write this function myself, but in any case I think I would sometime use offset to be -0.5 so that the bin centers correspond to integer values. Or, even though so far it never happened, in a more general way this generic axis being added could have a regular binning but not corresponding to integer values (e.g. a pt axis). This would require some generalisation, but maybe not to be dealt with by this PR